docs/scripts/formattedTSDemos.js

```diff
@@ -117,7 +117,14 @@ async function transpileFile(tsxPath, program, ignoreCache = false) {
         return undefined;
       },
     });
-    const codeWithPropTypes = typescriptToProptypes.inject(propTypesAST, code);
+    const codeWithPropTypes = typescriptToProptypes.inject(propTypesAST, code, {
+      shouldInclude: ({ prop }) => {
+        if (prop.jsDoc && prop.jsDoc.includes('@typescript-to-proptypes-ignore')) {
```

Comment:
@flaviendelangle What do you think about this? 🤔 
I got annoyed by overly complex proptypes being generated an in the case of `AutocompleteDatePicker` it was always failing the `options[0].isUTC` validation... 🤦

Reply:
I think that we should drop all the proptypes on the demos :laughing: 
But yes, that's a nice first step for sure !